test/clj/puppetlabs/trapperkeeper/testutils/webserver.clj

```diff
@@ -55,3 +58,25 @@
      ~port-var
      {}
      ~@body))
+
+(defn get-jetty-mbean-object-names
+  "TODO SPLAIN"
+  []
+  (let [mbeanserver (ManagementFactory/getPlatformMBeanServer)]
+    (.queryNames mbeanserver
+                 nil
+                 (ObjectName. "org.eclipse.jetty.jmx:type=mbeancontainer,*"))))
```

Comment:
Oh, I didn't realize we'd already introduced a dependency on that clojure lib.  Yes, we should definitely use it.